http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/SwaggerRoutes.scala

```diff
@@ -9,11 +9,9 @@ import akka.stream.scaladsl.Flow
 import akka.util.ByteString
 import org.broadinstitute.dsde.workbench.leonardo.config.SwaggerConfig
 
-trait SwaggerRoutes {
+class SwaggerRoutes(swaggerConfig: SwaggerConfig) {
   private val swaggerUiPath = "META-INF/resources/webjars/swagger-ui/2.2.5"
```

Comment:
Not for this PR, but latest swagger UI is 3.22

https://github.com/swagger-api/swagger-ui/releases?after=v3.22.2
https://swagger.io/tools/swagger-ui/

Reply:
3.25.0 it is! https://github.com/swagger-api/swagger-ui/releases :)